sys/include/net/gcoap.h

```diff
@@ -446,6 +446,16 @@ extern "C" {
  */
 #define GCOAP_PAYLOAD_MARKER    (0xFF)
 
+/**
+ * @ingroup net_gcoap_conf
+ * @brief   Flag to delay initialization until explicit command
```

Comment:
What explicit command? Something like `..delay initilization until @ref gcoap_init() is explicitly called.` might be clearer.

Reply:
OK, I'll include this suggestion as I rework with a different name, as suggested by @miri64.